HDMI CEC drivers
Enable support for HDMI CEC (Consumer Electronics Control),
which is an optional HDMI feature.

Say Y when you have an HDMI receiver, transmitter or a USB CEC
adapter that supports HDMI CEC.
Depends: MEDIA_SUPPORT && !MEDIA_SUPPORT_FILTER
---
--- /var/tmp/portage/sys-kernel/gentoo-sources-5.19.17/image/usr/src/linux-5.19.17-gentoo//.config.orig	2022-10-31 13:57:59.705695527 -0400
+++ /var/tmp/portage/sys-kernel/gentoo-sources-5.19.17/image/usr/src/linux-5.19.17-gentoo//.config	2022-11-01 16:19:55.774610338 -0400
@@ -5301,22 +5301,12 @@
 CONFIG_RC_LOOPBACK=m
 CONFIG_RC_XBOX_DVD=m
 CONFIG_CEC_CORE=m
-CONFIG_CEC_NOTIFIER=y
-CONFIG_CEC_PIN=y
 
 #
 # CEC support
 #
 CONFIG_MEDIA_CEC_RC=y
-# CONFIG_CEC_PIN_ERROR_INJ is not set
-CONFIG_MEDIA_CEC_SUPPORT=y
-CONFIG_CEC_CH7322=m
-# CONFIG_CEC_CROS_EC is not set
-CONFIG_CEC_GPIO=m
-CONFIG_CEC_SECO=m
-CONFIG_CEC_SECO_RC=y
-CONFIG_USB_PULSE8_CEC=m
-CONFIG_USB_RAINSHADOW_CEC=m
+# CONFIG_MEDIA_CEC_SUPPORT is not set
 # end of CEC support
 
 CONFIG_MEDIA_SUPPORT=m
